admin/javascript/LeftAndMain.Preview.js

```diff
@@ -607,8 +622,13 @@
 		$('.cms-preview.column-hidden').entwine({
 			onmatch: function() {
 				$('#preview-mode-dropdown-in-content').show();
+				// Alert the user as to why the preview is hidden
+				if ($('.cms-preview .result-selected').hasClass('font-icon-columns')) {
+					statusMessage("Screen too small to show site preview in split mode", "error");
```

Comment:
Yes, that's a good spot. 

This will need to go into https://github.com/silverstripe/silverstripe-framework/blob/3/admin/javascript/lang/src/en.js